### PR TITLE
ENTERPRISE-366 : Overall Client Library Code Example Updates

### PR DESCRIPTION
**Pull request by wootencl**
Completion of [ENTERPRISE-366](https://pokitdok.atlassian.net/browse/ENTERPRISE-366) ticket which entailed the updating of the overall client library code examples.

### CODE REVIEW

File: source/includes/_providers.md [hunk 1]
```diff
@@ -130,7 +130,11 @@ curl -i -H "Authorization: Bearer $ACCESS_TOKEN" https://platform.pokitdok.com/a
 ```
 
 ```python
+<<<<<<< HEAD
```
Comment (jeffhoek) [inline source/includes/_providers.md]: lil cleanup needed here? ;)

File: source/includes/_providers.md [hunk 2]
```diff
@@ -130,7 +130,11 @@ curl -i -H "Authorization: Bearer $ACCESS_TOKEN" https://platform.pokitdok.com/a
...
+client.providers(zipcode='29307', specialty='rheumatology', radius='20mi')
+=======
 pd.providers(zipcode='29307', specialty='rheumatology', radius='3mi')
+>>>>>>> df8fc6212c607697dc2f22d05ef933dd49c3ef54
```
Comment (jeffhoek) [inline source/includes/_providers.md]: lil cleanup needed here? ;)

**Comment by jeffhoek** [comment]
couple minor cleanup issues with git merge artifact otherwise 👍

**Comment by wootencl** [comment]
Whoops! Completely missed that. Good catch. Now fixed.

**Pull request merged**